src/vconsole/vconsole-setup.c

```diff
@@ -535,13 +538,21 @@ static int find_source_vc(char **ret_path, unsigned *ret_idx) {
                         RET_GATHER(err, r);
                         continue;
                 }
+
                 r = verify_vc_kbmode(fd);
                 if (r < 0) {
                         log_debug_errno(r, "Failed to check VC %s keyboard mode: %m", path);
                         RET_GATHER(err, r);
                         continue;
                 }
 
+                r = verify_vc_display_mode(fd);
+                if (r < 0) {
+                        log_debug_errno(r, "Failed to check VC %s display mode: %m", path);
+                        RET_GATHER(err, r);
```

Comment:
Please merge this with the log_debug_errno above

Reply:
I'm not sure to understand. The previous check is for the keyboard mode whereas this one is for the display mode. How can these be merged ?

Reply:
RET_GATHER(err, log_debug_errno(r, ...))

Reply:
ah ok. Since none of the rest of the code do that currently, can't this be done later in a separate PR ?

Reply:
This should be addressed by the second patch I just pushed.